flink-tests/src/test/java/org/apache/flink/test/recovery/SimpleRecoveryFailureRateStrategyITBase.java

```diff
@@ -43,7 +43,7 @@ private static Configuration getConfiguration() {
 		config.setString(RestartStrategyOptions.RESTART_STRATEGY, "failure-rate");
 		config.setInteger(RestartStrategyOptions.RESTART_STRATEGY_FAILURE_RATE_MAX_FAILURES_PER_INTERVAL, 1);
 		config.setString(RestartStrategyOptions.RESTART_STRATEGY_FAILURE_RATE_FAILURE_RATE_INTERVAL, "1 second");
-		config.setString(RestartStrategyOptions.RESTART_STRATEGY_FAILURE_RATE_DELAY, "100 ms");
+		config.setString(RestartStrategyOptions.RESTART_STRATEGY_FAILURE_RATE_DELAY, "0 s");
```

Comment:
why are we reducing the delay here? This doesn't seem strictly necessary.

Reply:
This is true. It speeds up the test though. I'll add it as a separate hotfix commit.